### ISSUE TITLE
Quest: Sorlof's Booty

### ISSUE THREAD

**Issue by TheRealYasri** [SunwellTracker/issues#332]
Description: https://db.darkwizard.pl/?quest=11529
Mob: https://db.darkwizard.pl/?npc=24914
Item Spawner: https://db.darkwizard.pl/?spell=45070
Quest Drop: https://db.darkwizard.pl/?item=34468

How it works: Mob is not visible from the ship to shot at.

How it should work: Shot the cannon from the ship to kill mob, once dead jump from ship to loot body on land.

I grouped with someone else and just tank and spanked the mob. But you only get one quest item per kill even in group. Would be good to give to all in group if on the same quest.

Source (you should point out proofs of your report, please give us some source):  Bug not blizzlike thing.

**Comment by Kokaina** [comment]
Duplicate.

**Comment by TheRealYasri** [comment]
Can you point me at the bug this is a duplicate of. I did a search for the quest name before posting and did not see any for it.
